doc/api/inspector.md

```diff
@@ -180,8 +182,32 @@ session.post('Profiler.enable', () => {
 });
 ```
 
+### Heap Profiler
+
+Here's a simple example showing how to use the [heap profiler][]:
+
+```js
+const inspector = require('inspector');
+const fs = require('fs');
+const session = new inspector.Session();
+
+session.connect();
+
+fs.writeFileSync('profile.heapsnapshot', '');
+
+session.on('HeapProfiler.addHeapSnapshotChunk', function(m) {
+  fs.appendFileSync('profile.heapsnapshot', m.params.chunk);
+});
+
+session.post('HeapProfiler.takeHeapSnapshot', null, function(err, r) {
+  console.log('Runtime.takeHeapSnapshot done:', err, r)
+  session.disconnect();
+});
+```
+
 [`'Debugger.paused'`]: https://chromedevtools.github.io/devtools-protocol/v8/Debugger#event-paused
 [`EventEmitter`]: events.html#events_class_eventemitter
 [`session.connect()`]: #inspector_session_connect
 [CPU Profiler]: https://chromedevtools.github.io/devtools-protocol/v8/Profiler
 [Chrome DevTools Protocol Viewer]: https://chromedevtools.github.io/devtools-protocol/v8/
+[heap Profiler]: https://chromedevtools.github.io/devtools-protocol/v8/HeapProfiler
```

Comment:
Lowercase P in Profiler?

Reply:
case doesn't seem to matter here, but I uppercased both profilers consistently, in text and in link.